documents/affiliated_package_review_guidelines.md

```diff
@@ -88,7 +107,9 @@ Note that general is not necessary better than specific, it’s just a way to ma
 
 ### Testing ('testing')
 
-Note: In our terminology, “tests” refer to those that can be run in an automated way, and we do not consider examples that need to be run and/or checked manually to be acceptable as “tests”
+In our terminology, “tests” refer to those that can be run in an automated way,
+and we do not consider examples that need to be run and/or checked manually to
+be acceptable as “tests”
```

Comment:
```suggestion
be acceptable as the primary way of satisfying “tests”
```
because sometimes it's ok to have a few manual tests if they are clear/simple.